### PR TITLE
Migrate to `sbt-ci-release`

### PR DESCRIPTION
**Pull request by guizmaii**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,51 +18,47 @@ jobs:
...
+          java-version: '8'
+          check-latest: true
+      - run: sbt ci-release
+        env:
+          ENCRYPTION_PASSWORD: ${{ secrets.ENCRYPTION_PASSWORD }}
+          PULL_REQUEST: ${{ !!github.event.pull_request }}
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```
Comment (guizmaii) [inline .github/workflows/ci.yml:64]: TODO: Extract these values from the (deleted) encrypted files

**Review by juliano — APPROVED** [review]
LGTM! 🥳

**Comment by tOverney** [comment]
@juliano  does that mean that now #309 can be fixed? :)

**Comment by guizmaii** [comment]
@juliano Do we want to wait to have sbt-ci-release in zio-quill too before making any new release?
I know that zio-protoquill depends on zio-quill. That could justify the additional wait but I don't know well enough about these projects nor their history nor the users' needs (like the one from @tOverney) to make a decision

**Comment by juliano** [comment]
I believe it's the sensible thing to do in order to keep the consistency among the versions of zio-quill and zio-protoquill. However, it would be a possibility to release a super-minor-minor version only for protoquill in the meantime? This way the new features become available while we decide to do without affecting the main releases much. Wty?

**Comment by guizmaii** [comment]
I'm good with this plan 🙂

**Pull request merged**